src/Core/include/config_core.h

```diff
@@ -0,0 +1,13 @@
+#pragma once
```

Comment:
Accidentially committed?

Reply:
Yes, it will be removed in https://github.com/ClickHouse/ClickHouse/pull/17435